webpack.config.js

```diff
@@ -1,38 +1,38 @@
+// The path to the CesiumJS source code
+const cesiumSource = 'node_modules/cesium/Source';
+const cesiumWorkers = '../Build/Cesium/Workers';
+const CopyWebpackPlugin = require('copy-webpack-plugin');
 const path = require('path');
-
 const webpack = require('webpack');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
-const CopyWebpackPlugin = require('copy-webpack-plugin');
 
-module.exports = [{
-    mode: 'development',
+module.exports = {
     context: __dirname,
     entry: {
         app: './src/index.js'
     },
     output: {
-        filename: '[name].js',
-        path: path.resolve(__dirname, 'dist')
+        filename: 'app.js',
+        path: path.resolve(__dirname, 'dist'),
+        sourcePrefix: ''
     },
-    devtool: 'eval',
-    node: {
-        // Resolve node module use of fs
-        fs: "empty",
-        Buffer: false,
-        http: "empty",
-        https: "empty",
-        zlib: "empty"
+    amd: {
+        // Enable webpack-friendly use of require in Cesium
+        toUrlUndefined: true
     },
     resolve: {
-        mainFields: ['module', 'main']
+        alias: {
+            cesium: path.resolve(__dirname, cesiumSource)
+        },
+        mainFiles: ['module', 'main', 'Cesium']
```

Comment:
Sorry - I corrected my comment - I believe this needs to be `[ 'index', 'Cesium' ]`.